scripts/build/macos-standalone.sh

```diff
@@ -91,41 +100,58 @@ EOF
 hdiutil detach "$mount_dir"
 
 # Convert to read-only
-hdiutil convert "$package_path" -format UDZO -o "$dmg_path"
+hdiutil convert "$staging_dmg_path" -format UDZO -o "$dmg_path"
 
 # Sign disk image
 codesign --force --sign "$codesign_identity" "$dmg_path"
 
 echo "Disk image created at $dmg_path"
 
-# Notarize disk image; notarizes embedded app bundle as well
+# Notarize disk image and package installer; notarizes embedded app bundle as well
 if [ "$notarize" = "true" ]; then
     private_key_path="$temp_dir/firezone-api-key.p8"
     base64_decode "$API_KEY" "$private_key_path"
 
-    # Submit app bundle to be notarized. Can take a few minutes.
-    # Notarizes embedded app bundle as well.
+    # Submit DMG to be notarized. Can take a few minutes. Notarizes embedded app bundle as well.
     xcrun notarytool submit "$dmg_path" \
         --key "$private_key_path" \
         --key-id "$API_KEY_ID" \
         --issuer "$ISSUER_ID" \
         --wait
 
-    # Clean up private key
-    rm "$private_key_path"
-
     # Staple notarization ticket to app bundle
     xcrun stapler staple "$dmg_path"
 
     # Verify notarization
     xcrun stapler validate "$dmg_path"
 
-    echo "Disk image notarized!"
```

Comment:
Not sure if it matters, but it looks like this echo message was left out of the change.  Was that intentional?

Reply:
Good catch fixed